android/src/main/kotlin/com/shatsy/admobflutter/AdmobBanner.kt

```diff
@@ -5,6 +5,9 @@ import android.view.View
 import com.google.android.gms.ads.AdRequest
 import com.google.android.gms.ads.AdSize
 import com.google.android.gms.ads.AdView
+import com.google.android.gms.ads.doubleclick.AppEventListener
+import com.google.android.gms.ads.doubleclick.PublisherAdRequest
+import com.google.android.gms.ads.doubleclick.PublisherAdView
```

Comment:
I've only worked with AdSense and Admob. How does DoubleClick fit into the family? I don't see a gradle dependency addition. I presume these classes are part of the same SDK?

Reply:
Hey @kmcgill88 Doubleclick is AdManager(Former DFP), the SDK is the same. DFP gives us a little bit more power and flexibility than AdMob and also We use same adserver for App/Web.

Speaking with our Google AM she said that we can use AdMob libraries for requesting AdManager AdUnits.

So I guess that we can Use Those views to also request AdMob.